### ISSUE TITLE
Chandra.Time build fails on linux64

### ISSUE THREAD

**Issue by javierggt** [sot/skare3#821]
**Description**

Chandra.Time build fails on linux64 with error:
```
x86_64-conda-linux-gnu-cc: error trying to exec 'cc1plus': execvp: No such file or directory
```

**To Reproduce**

The error is hard to catch in the workflow, but I reproduced it on HEAD:
```
conda create -y -n ska-builder setuptools_scm gitpython conda-build jinja2 pyyaml python=3.8 numpy
conda activate ska-builder
conda install -y jinja2 pyyaml packaging gitpython
conda install -y \
    -c https://ska:${CONDA_PASSWORD}@cxc.cfa.harvard.edu/mta/ASPECT/ska3-conda/masters \
    -c defaults -c conda-forge skare3_tools
cd /proj/sot/ska/jgonzalez/git/skare3
./ska_builder.py --tag 4.0.1 Chandra.Time
```

**Comment by taldcroft** [comment]
I guess it is probably not helpful, but I did confirm that `python setup.py build_ext --inplace` successfully builds on HEAD.

**Comment by javierggt** [comment]
Yes, the issue is that conda tries to be smart about compilers to be consistent, plus the compilers are installed in the build environment, so the build does not see the system compilers.

**Comment by jeanconn** [comment]
I still get a bunch of warnings, but on linux it looks to me like it builds if I replace
`gcc_linux-64==8.4.0 # [linux]` with `gxx_linux-64==8.4.0 # [linux]` .  I guess I thought gcc_linux provided the c++ compiler but it looks like it does not.  We could also move to a new version etc I suppose.

**Comment by jeanconn** [comment]
Yeah, not super clear why it _stopped_ working, but maybe a different build of *another* package stopped getting the gxx package as an implicit conda dependence in the build environment or some such?
